components/youtube_video/youtube_video.jsx

```diff
@@ -213,13 +124,15 @@ export default class YoutubeVideo extends React.PureComponent {
                 />
             );
         } else {
+            const images = metadata.images[0];
+
             content = (
-                <div className='embed-responsive embed-responsive-4by3 video-div__placeholder'>
+                <div className='embed-responsive video-div__placeholder'>
                     <div className='video-thumbnail__container'>
                         <img
```

Comment:
Using `ExternalImage` but does it add any value here?

> you can remove the hasImageProxy prop since it's no longer used by YoutubeVideo

Nice catch thanks. I forgot to check props. Looking at them again i was able to remove few more. have another look